diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/DataClassesRule.kt

```diff
@@ -0,0 +1,73 @@
+package org.cqfn.diktat.ruleset.rules
```

Comment:
```suggestion
package org.cqfn.diktat.ruleset.rules.classes
```
Let's try to place rules in different packages when it's logically possible